### PR TITLE
Master rel fields fix aab

### PR DESCRIPTION
**Pull request by aab-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/src/views/helpers/standard_view_props.js [hunk 1]
```diff
@@ -24,6 +24,7 @@ export const standardViewProps = {
     context: { type: Object },
     domain: { type: Array },
     fields: { type: Object, elements: Object },
+    relatedModels: { type: Object, elements: Object },
```
Comment (ged-odoo) [inline addons/web/static/src/views/helpers/standard_view_props.js:27]: optional

File: addons/web/static/src/webclient/actions/action_service.js [hunk 2]
```diff
@@ -185,7 +185,11 @@ function makeActionManager(env) {
      * with a unique jsId.
      */
     function _preprocessAction(action, context = {}) {
-        action._originalAction = JSON.stringify(action);
+        try {
+            action._originalAction = JSON.stringify(action);
+        } catch (_e) {
+            // do nothing, the action might simply not be stringified (e.g. if it contains cycles)
```
Comment (ged-odoo) [inline addons/web/static/src/webclient/actions/action_service.js:191]: `the action might simply not be stringified` => `the action might simply not be serializable`

**Comment by aab-odoo** [comment]
robodoo r+

**Comment by aab-odoo** [comment]
robodoo r-

**Comment by aab-odoo** [comment]
robodoo rebase-ff

**Comment by ged-odoo** [comment]
robodoo r+

**Pull request closed without merging**